### PR TITLE
Font/karla bold

### LINKED ISSUE #46: Import bold weight Karla font
Figma specifies a 700 weight font for some of the designs. Currently only 400 and 600 are imported.

Should we add a 700 weight Karla.woff?

image

### PR DESCRIPTION
**Pull request by mmkobylin**
Karla bold imported and updated in index.html

### CODE REVIEW

**Comment by shanesc** [comment]
Hi Marta, a couple of things when I tested:

1. I think the same `@font-face` code needs to be added to the `preview-head.html` file to work in Storybook. This is due to the way Storybook loads assets (https://storybook.js.org/docs/react/configure/story-rendering#adding-to-head)
image

2. The code looks good but I think the `.woff` files are not the correct weight. I'm seeing no difference between 600, 700, and 800 in Storybook components. Let me know if you are seeing the same.

Hopefully that makes sense but ping me on Slack if you want to discuss.

File: public/index.html [hunk 1]
```diff
@@ -40,6 +40,15 @@
...
       }
+      @font-face {
+        font-display: swap;
+        font-family: 'Karla';
+        font-style: normal;
+        font-weight: 800;
+        src: local('Karla Bold'), local('Karla-Bold'),
+          url(/assets/fonts/Karla-Bold.woff2) format('woff2'),
+          url(/assets/fonts/Karla-Bold.woff) format('woff');
+      }
```
Comment (shanesc) [inline public/index.html:51]: This block of code needs to be added to `.storybook/preview-head.html` as well. See https://storybook.js.org/docs/react/configure/story-rendering#adding-to-head

**Review by shanesc — CHANGES_REQUESTED** [review]
Please check the `.woff` files are correct as they're not rendering bold on Storybook. Also add the `@font-face` to `preview-head.html`

**Comment by mmkobylin** [comment]
Hi, I have changed the files and tested them - there is a difference between using font-weight: 600 and 700. Please double check if everything looks correct.

**Review by shanesc — CHANGES_REQUESTED** [review]
Hi Marta, looks great and is working for me!

Just one last thing, can you please delete the `Karla-Large.woff` and `Karla-Large.woff2` files as they're not being used. Once that's done it's ready to be merged 👍

**Review by shanesc — APPROVED** [review]
Looks good! Squash and Merge 👍

**Pull request merged**